.github/workflows/event-consumer.yml

```diff
@@ -0,0 +1,53 @@
+name: Event consumer
+
+on:
+  push:
+    branches:
+      - "**"
+    paths:
+      - eventconsumer/**
+      - commoneventconsumer/**
+      - project/**
+      - build.sbt
+      - .github/workflows/event-consumer.yml
+  workflow_dispatch:
+
+# allow queued workflows to interrupt previous runs
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: corretto
+          cache: sbt
+          java-version: 11
+
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: eu-west-1
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+
+      - name: Compile, test and assembly
+        run: sbt "project eventconsumer" "compile" "test" "assembly"
+
+      - name: Upload to riff-raff
+        uses: guardian/actions-riff-raff@v2
+        with:
+          projectName: mobile-n10n:eventconsumer
+          configPath: eventconsumer/riff-raff.yaml
+          contentDirectories: |
+            eventconsumer:
+              - eventconsumer/target/scala-2.13/eventconsumer.jar
```

Comment:
Sometimes I've had differences between what ran locally and what came out in github actions, for the version of scala, ie. 2.12 locally vs. 2.13 in GHA, and those could change in theory.
Sometimes to fix this, I've added something like this:

```
      - name: copy jar to root dir
        run: cp eventconsumer/target/scala-*/eventconsumer.jar .
``` 
then in the riffraff action, provided the path at the root directory

Reply:
Great idea - I've added this extra step :)